_IBC/uptick-iris.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "../ibc_data.schema.json",
+  "chain_1": {
+    "chain_name": "uptick",
+    "client_id": "07-tendermint-2",
+    "connection_id": "connection-0"
+  },
+  "chain_2": {
+    "chain_name": "iris",
```

Comment:
same here, irisnet should be chain 1, and uptick as chain 2 (and file renamed accordingly)

Reply:
ok